bindings/lua/Cargo.toml

```diff
@@ -29,9 +29,27 @@ rust-version.workspace = true
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+
+[features]
+default = ["mlua/lua54"]
+lua54 = ["mlua", "mlua/lua54"]
+lua53 = ["mlua", "mlua/lua53"]
+lua52 = ["mlua", "mlua/lua52"]
+lua51 = ["mlua", "mlua/lua51"]
+luajit = ["mlua", "mlua/luajit"]
+vendored = ["mlua", "mlua/vendored"]
+
+# for disable default features
+[dependencies.mlua]
+version = "0.8"
+features = ["module"]
+default-features = false
+optional = true
+
+
 [lib]
 crate-type = ["cdylib"]
+name = "lopendal"
```

Comment:
Why using `lopendal`?

Reply:
because the `opendal-lua` name will produce `libopendal_lua.so`, it can not correct read by `luarocks-build-rust-mlua`, if it config module name as `opendal` in `luarocks`,  `libopendal.so` will not be found (not exist), if I set cargo build output name as `libopendal.so`, Maybe it will conflict with other library, because `libopendal.so` scope is too big. So I chose `lopendal` as lua module name, it clearly and not conflict with other library.

Reply:
No, it's not a bug I think.
```
opendal -> libopendal.so
```
this relationship is reasonable, but opendal project seems too complex that we need to work around it, I will rethink how to resolve this

Reply:
I believe that `opendal-lua`, not `opendal`, provides dynamic files. The so file only provides the API required by Lua. This naming convention is used for all bindings, such as `libopendal_c.so` and `libopendal_python.so`.

Please note that `opendal` only provides rust API, and `opendal-lua`'s crate type is `cdylib`.

Reply:
By the way, is it required that the so name must be the same with `rockspec.name`?

Reply:
Yeap, the problem is that the so name must be the same with rockspec.name

Reply:
Maybe we can name it as `lua-opendal` in Lua rocks module?

Reply:
sloved https://github.com/khvzak/luarocks-build-rust-mlua/pull/4